### ISSUE TITLE
False-positive on unusedArguments

### ISSUE THREAD

**Issue by romdevios** [nicklockwood/SwiftFormat#1696]
Parameter in this code counts as unused because placed on the next lines after the same named parameter created.

```swift
func someFunction(with parameter: Int) -> Int {
    let parameter = max(
        200,
        parameter
    )
    return parameter
}
```

Formatted to and not compiling anymore:

```swift
func someFunction(with _: Int) -> Int {
    let parameter = max(
        200,
        parameter
    )
    return parameter
}
```

**Comment by nicklockwood** [comment]
@romdevios thanks for reporting this, I'll get it fixed. A workaround for now seems to be to put the expression all on one line:

```swift
let parameter = max(200, parameter)
```

**Comment by nicklockwood** [comment]
@romdevios fixed in 0.53.10
